src/domain/accounts/services/src/dependencies.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use dill::CatalogBuilder;
+
+use crate::*;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub fn register_dependencies(catalog_builder: &mut CatalogBuilder, needs_indexing: bool) {
```

Comment:
Do we have any more domains level, where it's still not done like that?

Reply:
Added some similar stuff, but not for domains, but for `kamu-cli` entities.

The kamu namespace remains, which needs to be split up, as it has a huge number of dependencies.

Partly done.